changelog.md

```diff
@@ -1,3 +1,24 @@
+# Version 0.4.0
+
+## Features
+
+* [METRONOME-190](https://jira.mesosphere.com/browse/METRONOME-190) Added authorized launch queue
+* [METRONOME-194](https://jira.mesosphere.com/browse/METRONOME-194) Support FORBID Concurrency Policy
+
+## Bugs and Tracking
+
+* [METRONOME-100](https://jira.mesosphere.com/browse/METRONOME-100) Metronome Restarts causes duplication of jobruns
+* [METRONOME-191](https://jira.mesosphere.com/browse/METRONOME-191) Implement Start Deadline Timeout
```

Comment:
Please make it consistent with respect to capital letters usage throughout the diff.